### PR TITLE
Update running_moments.py

### PR DESCRIPTION
**Pull request by auroradh**
add function to normalize xarray dataset

### CODE REVIEW

**Comment by Sosnowsky** [comment]
@gregordecristoforo @sajidah-ahmed I believe this function should be moved to Sajidah's repo, since it deals with datasets produced by that code. do you agree?

**Comment by gregordecristoforo** [comment]
Yes, I agree. https://github.com/sajidah-ahmed/cmod_functions is a much better place for it.

**Comment by sajidah-ahmed** [comment]
I think my concern earlier was that cmod_functions would end up having a dependency on fpp-analysis-tools. It seems like the function still relies on fpp-analysis-tool because it's using run_norm. The idea behind cmod_functions is that if someone wishes to just pull data from the MDS tree (without analysing time series according to the fpp), then they can use that without having to download an extra package. Since run_norm_ds is the only function that requires fpp-analysis-tools, I feel that it makes sense to keep it on fpp-analysis-tools with the caveat added to the doc string that one would need to download cmod-functions is they choose to work with experimental data. What do you both think?

**Comment by sajidah-ahmed** [comment]
@gregordecristoforo didn't you put `run_norm` is `plasmapy` recently? If so, wouldn't it make sense to incorporate into `run_norm_ds`?

File: fppanalysis/running_moments.py [hunk 1]
```diff
@@ -149,3 +149,56 @@ def window_radius(cut_off_freq, time):
     samples = ((t_run_mean / dt) - 1) / 2
 
     return int(samples)
+
+def run_norm_ds(ds, radius, **kwargs):
```
Comment (Sosnowsky) [inline fppanalysis/running_moments.py:153]: kwargs not used? If so, remove it

**Comment by Sosnowsky** [comment]
That's a good point @sajidah-ahmed , I agree its nice to keep cmod-functions independent of fpp-analysis. 
We can keep this function here then, if you are fine with that.
@auroradh I just made a small comment, if you resolve we cna merge

**Comment by gregordecristoforo** [comment]
@Sajidah yes run_norm is available in plasmapy. If you don't mind having plasmapy as a dependency we could merge it here. However, keeping the number of dependencies low is always a good idea.

File: fppanalysis/running_moments.py (hunk 1 above)
Comment (auroradh) [reply fppanalysis/running_moments.py]: I changed it now, and it is ready to merge. However, it says that there still is a conflict, even though there is only new code added?

**Review by Sosnowsky — APPROVED** [review]

**Pull request merged**